core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -65,7 +62,7 @@
             TESTCONTAINERS_SESSION_ID_LABEL, SESSION_ID
     );
 
-    private static final DockerImageName TINY_IMAGE = DockerImageName.parse("alpine:3.5");
+    private static final DockerImageName RYUK_IMAGE = DockerImageName.parse("testcontainers/ryuk:0.3.1");
```

Comment:
make sure that `ResourceReaper.start` uses it as well

Reply:
Yes, it does.

Reply:
err.... does it? :D 

https://github.com/testcontainers/testcontainers-java/blob/6086239f791b74a88af578ae714dafb90cd5983d/core/src/main/java/org/testcontainers/utility/ResourceReaper.java#L92

Reply:
Err.. Yes?

Or do you mean reuses the same constant, rather than the same string?

Reply:
Ah, yes, sorry! I meant using the safe object ref, to keep it consistent (otherwise I bet we will forget to update one while updating another and cause double image downloads :D)

Reply:
Aha sure OK. I think I'll move the constant over to `ResourceReaper` and reference it here.